clang/lib/CIR/Dialect/Transforms/TargetLowering/Targets/LoweringPrepareX86CXXABI.cpp

```diff
@@ -0,0 +1,357 @@
+//====- LoweringPrepareX86CXXABI.cpp - Arm64 ABI specific code -------====//
+//
+// Part of the LLVM Project,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===------------------------------------------------------------------===//
+//
+// This file provides X86{_64, _32} C++ ABI specific code that is used during
+// LLVMIR lowering prepare.
+//
+//===------------------------------------------------------------------===//
+
+#include "../LowerModule.h"
+#include "../LoweringPrepareItaniumCXXABI.h"
```

Comment:
Should we change CMAKE for this dir so that we can just do `#include "LowerModule.h"` and others directly?

Reply:
I am not sure. I can see other `#include "../` pattern in the codes. More importantly, I feel good with the current style since I can have better and more explicit understanding to the structure. Especially we have some duplicated header names (e.g., `ABIInfo.h`).

Reply:
> we have some duplicated header names

good point